includes/class-amp-travel-blocks.php

```diff
@@ -508,6 +519,9 @@ public function render_block_travel_featured( $attributes ) {
 
 		// The count has to be 6 to fill the grid.
 		if ( count( $locations ) !== self::FEATURED_LOCATIONS_COUNT ) {
+			if ( $is_server_side_render ) {
+				return __( 'Not enough featured locations found. Please add at least six "Locations" terms, select an image, and check "Featured destination."', 'travel' );
+			}
```

Comment:
This is an alternative to the placeholder that used to be in the `edit()` function. It's not really ideal.

It'd be nice if there was a way to detect if the `<ServerSideRender>` component didn't render anything, so this placeholder text didn't have to be in the `render_callback`. 

Maybe there's a way to do that.

Reply:
It should be sufficient for the purposes of the demo.

Reply:
Nice, thanks